storage/scylladb_chronicle.h

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (C) 2020 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+#ifndef STORAGE_SCYLLADB_CHRONICLE_H_
+#define STORAGE_SCYLLADB_CHRONICLE_H_
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "cclient/request/find_transactions.h"
+#include "cclient/request/get_inclusion_states.h"
+#include "cclient/request/get_trytes.h"
+#include "cclient/response/find_transactions.h"
+#include "cclient/response/get_inclusion_states.h"
+#include "cclient/response/get_trytes.h"
+#include "common/model/transaction.h"
+#include "scylladb_client.h"
+#include "utils/containers/hash/hash8019_queue.h"
+
+/**
+ * @file storage/scylladb_chronicle.h
+ * @brief c implement ch.
+ */
+
+/**
+ * @brief connect to ScyllaDB node and create table
+ *
+ * @param[in] service ScyllaDB db client service
+ * @param[in] need_truncate true : clear all data, false : keep old data
+ * @param[in] keyspace_name keyspace name the session should use
+ *
+ * @return
+ * - SC_OK on success
+ * - non-zero on error
+ */
+status_t db_chronicle_keyspace_init(const db_client_service_t* service, bool need_truncate, const char* keyspace_name);
+
+/**
+ * @brief insert transactions into bundle table
+ *
+ * @param[in] service ScyllaDB client service
+ * @param[in] trytes input transaction trytes
+ *
+ * @return
+ * - SC_OK on success
+ * - non-zero on error
+ */
+status_t db_chronicle_insert_transaction(const db_client_service_t* service, const tryte_t* hash,
+                                         const tryte_t* trytes);
+
+/**
+ * @brief get transaction hash by given bundle
+ *
+ * @param[in] service ScyllaDB client service
+ * @param[out] res result hash queue for transaction hash
```

Comment:
Organize the order of parameters here. `param[out]` should come after `param[in]`.